aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -179,16 +179,17 @@ void fbgemm_spmdm_report_error_(
           0 <= idx && idx < N,
           "Index ",
           i,
-          " is out of bounds: ",
+          " of input takes value ",
           idx,
-          ", range 0 to ",
-          N);
+          " which is not in the valid range [0, ",
+          N,
+          "[");
```

Comment:
nit: exclusive bracket for N

```suggestion
          " which is not in the valid range [0, ",
          N,
          ")");
```

Reply:
Of course, just fixed.